api/capi/include/nntrainer.h

```diff
@@ -144,6 +144,20 @@ int ml_nnmodel_train_with_generator(ml_nnmodel_h model,
  */
 int ml_nnmodel_destruct(ml_nnmodel_h model);
 
+/**
+ * @brief Get the summary of the neural network model.
+ * @details Use this function to get the summary of the neural network model.
```

Comment:
How about adding that the summary will be for the model and layers (and not tensors for now as the API does not expose tensor details).

Reply:
Done